draft-ietf-mmusic-ice-sip-sdp-38.xml

```diff
@@ -1341,41 +1341,114 @@ It is outside of the scope of ICE for it to act as a tool for "working around" S
+                address and port matching the values in the "m=" and "c=" lines or rtcp attribute, the ALG uses that
+                binding instead of creating a new one.
+              </t>
+              <t>
+                If the ALG does not already have a binding, it creates a new one and modifies the SDP, rewriting
+                the "m=" and "c=" lines and rtcp attribute.
+              </t>
+            </list>
+          </t>
+        </list>
+      </t>
+      <t>
+        Unfortunately, many ALGs are known to work poorly in these corner cases.
+        ICE does not try to work around broken ALGs, as this is outside the scope of its functionality.
+        ICE can help diagnose these conditions, which often show up as a mismatch between the set of candidates and
+        the "m=" and "c=" lines and rtcp attributes. The ice-mismatch attribute is used for this purpose.
+      </t>
+      <t>
+        ICE works best through ALGs when the signaling is run over TLS.
+        This prevents the ALG from manipulating the SDP messages and interfering with ICE operation.
+        Implementations that are expected to be deployed behind ALGs SHOULD provide for TLS transport of the SDP.
+      </t>
+      <t>
+        If an SBC is SIP aware but not ICE aware, the result depends on the behavior of the SBC.
+        If it is acting as a proper Back-to-Back User Agent (B2BUA), the SBC will remove any SDP attributes
+        it doesn&#8217;t understand, including the ICE attributes. Consequently, the call will appear to both
+        endpoints as if the other side doesn&#8217;t support ICE. This will result in ICE being disabled, and
+        media flowing through the SBC, if the SBC has requested it. If, however, the SBC passes the ICE attributes
+        without modification, yet modifies the default destination for media (contained in the "m=" and "c=" lines
+        and rtcp attribute), this will be detected as an ICE mismatch, and ICE processing is aborted for the call.
+        It is outside of the scope of ICE for it to act as a tool for "working around" SBCs.
+        If one is present, ICE will not be used and the SBC techniques take precedence.
+      </t>
+    </section>
     <section title="Security Considerations">
+      <t>
+        The generic ICE security considierations are defined in <xref target="RFC8445"/>, and the 
+        generic SDP offer/answer security considerations are defined in <xref target="RFC3264"/>. The 
+        security considerations also apply to implementations of this document.
+      </t>
+      <section title="IP Address Privacy">
+        <t>
+          In some cases, e.g., for privacy reasons, an agent may not want to reveal the related
+          address and port. In this case the address MUST be set to "0.0.0.0" (for IPv4 candidates)
+          or "::" (for IPv6 candidates) and the port to zero.
+        </t>
```

Comment:
port zero would make m=line unusable isn't it ? did you mean port '9' instead ?

Reply:
Yes. I will fix it in section 5.1 too, because I copy/pasted the text from there (as requested by Benjamin).